src/components/Layout/ScrollableContent.tsx

```diff
@@ -24,7 +24,7 @@ export function ScrollableContent(props: { children: ReactNode; virtualized?: bo
 
   return createPortal(
     !virtualized ? (
-      children
+      <div css={Css.h100.addIn("&:after", Css.contentEmpty.db.h2.$).$}>{children}</div>
```

Comment:
Suggestion: add an in-source comment? Granted ianae but I wouldn't immediately intuit what this is for.